src/classes/TDTM_TriggerHandler.cls

```diff
@@ -181,15 +182,19 @@ public with sharing class TDTM_TriggerHandler {
     
     private void runAsync(TDTM_Runnable classToRun, String classToRunName, List<Sobject> newList, 
     List<Sobject> oldList, TDTM_Runnable.Action thisAction, Schema.DescribeSobjectResult describeObj) {
-       Map<Id,Sobject> nm;
-       if(newlist != null) nm = new Map<Id,Sobject>(newlist);
-       else nm = new Map<Id,Sobject>();
+       set<Id> setNewId;
```

Comment:
The only difference that I see in this whole section is that now we are passing nulls to runFutureNonStatic if newlist or oldlist are null.  Does new set<Id>(om.keySet()) really give you something different from om.keySet()?!?